C/CUDA/CUDA_SDK@13.0/build_tarballs.jl

```diff
@@ -13,4 +13,11 @@ platforms = [Platform("x86_64", "linux"),
              Platform("aarch64", "linux"),
              Platform("x86_64", "windows")]
 
-build_sdk(name, version, platforms; static=false)
+full_platforms = Platform[]
+for platform in platforms
+    augmented_platform = deepcopy(platform)
+    augmented_platform["cuda"] = "$(version.major)"
+    push!(full_platforms, augmented_platform)
+end
```

Comment:
Why did you add the platform augmentation to the SDKs? I think this is now breaking the Ygg install of the SDKs when scripts try to build against the 13.0 SDK, because we aren't building in an environment that has the cuda platform in its string, we are building on a plain musl platform (https://github.com/JuliaPackaging/Yggdrasil/issues/12119).

Reply:
https://buildkite.com/julialang/yggdrasil/builds/23214#01991a5e-0ac5-459c-8a82-3dd74b927824